### ISSUE TITLE
[help] How to write a first set of row just after earch rotation (ex, column header)

### ISSUE THREAD

**Issue by franckOL** [iccicci/rotating-file-stream#109]
I try event rotated and rotation

**Comment by iccicci** [comment]
Happy for that @franckOL 😄 but it doesn't sound as an issue.

Do you have some troubles?

**Comment by franckOL** [comment]
Sure not an issue.
just trying to write a header in first row of each file, and can't find the way

**Comment by iccicci** [comment]
Hmm no, ther's no a way to do that: some more `write` could be called during the rotation procedure.

You could manipulate the file in the `rotated` event

**Comment by franckOL** [comment]
I try rotated event, I think a lot a thing are done in async way, because, my header is present but not a the first line.

**Comment by iccicci** [comment]
You need to maipulate the file after it is rotated (in the `rotated` event) you can't trust events to add the header with a `write`.

**Comment by franckOL** [comment]
oh, preprend the file with header :-)
I will try this as a post action

**Comment by iccicci** [comment]
Yep, please feel free to close the issue if this solves your poroblem.

**Comment by franckOL** [comment]
It works, need to read a quite large file.
```javascript
writeStream.on( 'rotated', async (filename) => {
    logger.error(filename)
    const data = await fs.readFile(filename)
    const fd = await fs.open(filename, 'w+')
    await fd.write(columnNameHeader)
    await fd.write(data)
    await fd.close()
})
```
